src/WebJobs.Script/Diagnostics/FileLoggingMode.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public enum FileLoggingMode
+    {
+        /// <summary>
+        /// Never log to file (the default).
+        /// </summary>
+        Never,
+        /// <summary>
+        /// Always log to file.
+        /// </summary>
+        Always,
```

Comment:
Always allows us to force file logging to happen if need be. Can be set via host.json